### DISCUSSION TITLE
multiblock machines don't work

### DISCUSSION THREAD

**Discussion by DrHleeb** [Rearth/Oritech#109] [Q&A]
hi, i downloaded the mod and built foundry and for some reason it ignores machine cores, at first i thought that better cores were needed but it still doesn't work, explain what i'm doing wrong
![javaw_rTKzSHRluA](https://github.com/user-attachments/assets/4644c894-4a07-4f0b-be29-b309363f606d)

**Comment by Rearth** [comment]
I'll need a few more details. Can you share a picture of the foundry? What are you trying to achieve? What makes you think it ignores the cores? where did you put the cores?

**Reply by DrHleeb** [reply]
i want to make steel ingots and i put all the cores on the highlighted places and the interface says ignored and it doesn't work
![javaw_dj9WXwh3gC](https://github.com/user-attachments/assets/cae8ade8-9694-4777-86dc-d82f30618ffe)
![javaw_Gi1oAP2L6f](https://github.com/user-attachments/assets/0331e8bd-41a0-487f-bd41-29ab3a195b19)
![javaw_OqOdt1yonM](https://github.com/user-attachments/assets/a674a708-46a5-401d-bd9d-c44e029bbcc0)
![javaw_WSNULZypwD](https://github.com/user-attachments/assets/28fbff04-f4c5-4a8d-8e9d-49fa38833fe0)

**Comment by Rearth** [comment]
Oh, now I see the issue. That's a very unfortunate text rendered there. However it's actually caused by inventory profiles next I believe. The ignored text has nothing to do with the machine, all is working well. It's just inventory profiles next (the mod) deciding to screw up the Oritech UI.

**Comment by Rearth** [comment]
Closing this for now
